apps/server/src/modules/tldraw/controller/tldraw.ws.ts

```diff
@@ -12,18 +17,55 @@ export class TldrawWs implements OnGatewayInit, OnGatewayConnection {
 
 	constructor(
 		private readonly configService: ConfigService<TldrawConfig, true>,
-		private readonly tldrawWsService: TldrawWsService
+		private readonly tldrawWsService: TldrawWsService,
+		private readonly logger: Logger
 	) {}
 
-	public handleConnection(client: WebSocket, request: Request): void {
+	async handleConnection(client: WebSocket, request: Request): Promise<void> {
 		const docName = this.getDocNameFromRequest(request);
-
+		const cookies = this.parseCookiesFromHeader(request);
```

Comment:
I thought about it, but then it would be best to call closeClientAndLogError() in it if we don't have jwt, so we would have to add it's all parameters in parseCookiesFromHeader() and i don't think it will be best styling option to do that.
We could of course add mentioned check right after line 26 but this check is done 3 lines below - in tldrawWsService.authorizeConnection() according to your suggestions.
I think it is enough to have check there, but if you don't agree - let me know.

Also i now moved cookie extraction after checking if we got docName and if tldraw feature is enabled. I think it is better placing for it.